src/setup/config.rs

```diff
@@ -1,27 +1,29 @@
-use std::{
-    collections::HashSet,
-    ffi::OsString,
-    fmt::Write,
-    fs,
-    net::{IpAddr, SocketAddr},
-    path::PathBuf,
-};
+use std::{ffi::OsString, fmt::Write, fs, path::PathBuf};
 
 use anyhow::Result;
 use serde::Deserialize;
 
-// Ziggurat's configuration directory and file. Caches are written to this directory.
+use crate::setup::node::NodeConf;
+
+/// Ziggurat's configuration directory.
 pub const ZIGGURAT_DIR: &str = ".ziggurat";
-// Configuration file with paths to start rippled.
+
+/// Ziggurat's Ripple's subdir
```

Comment:
We are missing a dot at the end of the comment. It seems like that's a convention here.
(Same thing on L14)

Reply:
Done